prep-html-syntax/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+
+ <h1 style="color: blue;">My favorite Start Wars character is Anakin!</h1>
+ <title>HTML Syntax</title>
+
+ <img src="https://anime-figure.com/static/images/full/9a235056db77e48e3b1822d995e1ac732cd7fd8a.jpg" alt="Anakin Skywalker">
+
+<p style="color: crimson;">Anakin completely rebuilds himself (even though he turns to the dark side...) </p>
+
+</body>
+
+</html>
```

Comment:
Hi Addy, be sure to divide your page into two specific sections. The `<head>` should open and close before the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>
  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>
</html>
```